oracle/tests/scripts/start_commands_jdbc.sh

```diff
@@ -0,0 +1,16 @@
+# This script makes the necessary setup to be able to compile pymqi on the agent machine
```

Comment:
Could we use a simple sh script instead of two? They look almost identical except that for jdbc we don't install libaio1

Reply:
Sure, good idea. It's more DRY now.